urnx/types.go

```diff
@@ -0,0 +1,15 @@
+package urnx
+
+import "github.com/google/uuid"
+
+const PREFIX = "urn"
+const URNLENGTH = 4
+
+// URN is an infratographer based URN consisting of a namespace, resource type and resource ID
+type URN struct {
+	Prefix       string
```

Comment:
`Prefix` isn't necessary here. The first part of a URN is always `urn`: https://www.rfc-editor.org/rfc/rfc8141.html#section-2

Reply:
✅ Dropped this from the URN object and just have it prefixed in the `String` func representation